bench/README.md

```diff
@@ -0,0 +1,55 @@
+# Benchmarks
```

Comment:
I think you need to add `bench` to the `skipDirs` list of the nimble package. Otherwise `nimble check` will complain that you have violated the rules for allowed package structure.

Alternatively, "bench" can put as a sub-directory inside "tests"